docs/ug/cli.md

```diff
@@ -31,7 +31,8 @@ The section below provides explanations for each of the flags.
 <div id="section-assets">
 
 **`--assets ASSETS_DIRECTORY`**: Specifies where to place assets for report generation.
-* Parameter: `ASSETS_DIRECTORY` The directory containing the assets files. A `favicon.ico` file can be placed here to customize the favicon of the dashboard.
+* Parameter: `ASSETS_DIRECTORY` The directory containing the assets files. A `favicon.ico` file can be placed here to customize the favicon of the dashboard,
+while a `title.md` file can be placed to customize the header of the report using [Markdown syntax](https://www.markdownguide.org/basic-syntax/).
```

Comment:
@sopa301 Noticed while passing: There should not be a line break here (in markdown files, we should not wrap lines to a specific length).

Reply:
@sopa301 Further comments:
1. There should be more details of this feature. For example, code for an example title component.
2. Doesn't feel like this is the right place to document this feature. Perhaps add as a separate section in https://reposense.org/ug/customizingReports.html ?

Reply:
Thanks for the review prof, I'll work on it!